docs/api/api-specification.md

```diff
@@ -0,0 +1,2170 @@
+        "subject": {
+          "reference": "Patient/3"
+        },
+        "item": [
+          {
+            "linkId": "0",
+            "text": "cough",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "1",
+            "text": "difficulty-breathing",
+            "answer": [
+              {
+                "valueBoolean": true
+              }
+            ]
+          },
+          {
+            "linkId": "2",
+            "text": "fever",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "3",
+            "text": "used-a-fever-reducer",
+            "answer": [
+              {
+                "valueBoolean": true
+              }
+            ]
+          },
+          {
+            "linkId": "4",
+            "text": "chills",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "5",
+            "text": "repeated-shaking-with-chills",
+            "answer": [
+              {
+                "valueBoolean": true
+              }
+            ]
+          },
+          {
+            "linkId": "6",
+            "text": "muscle-pain",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "7",
+            "text": "headache",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "8",
+            "text": "sore-throat",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "9",
+            "text": "new-loss-of-taste-or-smell",
+            "answer": [
+              {
+                "valueBoolean": true
+              }
+            ]
+          }
+        ],
+        "resourceType": "QuestionnaireResponse"
+      }
+    }
+  ],
+  "resourceType": "Bundle"
+}
+```
+</details>
+
+
+<a name="create"/>
+
+### Creating
+
+The API supports creating new monitorees.
+
+#### Extensions
+
+<a name="create-ext"/>
+
+Along with supporting the US Core extensions for race, ethnicity, and birthsex, Sara Alert includes four additional extensions for things specific to the Sara Alert workflows.
+
+Use `http://saraalert.org/StructureDefinition/preferred-contact-method` for specifying the monitorees Sara Alert preferred contact method (options are: `E-mailed Web Link`, `SMS Texted Weblink`, `Telephone call`, `SMS Text-message`, `Opt-out`, and `Unknown`).
+
+```json
+{
+  "url": "http://saraalert.org/StructureDefinition/preferred-contact-method",
+  "valueString": "E-mailed Web Link"
+}
+```
```

Comment:
But this formats well, interesting, since they both use ` ```json `

Reply:
Yeah it's bc this one's not nested in HTML